### PR TITLE
Adding security related questions to PR template

### PR DESCRIPTION
**Pull request by pzhou-splunk**
### Notes
- Copying over and re-wording the questions from the ProdSec questionnaire that would still be useful to explicitly address and have reviewed by a human
- All feedback/suggestions are welcome :)

### CODE REVIEW

**Review by jdemelo — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -24,6 +24,16 @@ Please ensure your pull request (PR) adheres to the following guidelines:
 - [ ] Documentation
 - [ ] Other (please describe): 
 
+## Security Considerations (REQUIRED)
```
Comment (rgil-splunk) [inline .github/pull_request_template.md:27]: should we add "checkboxes" like the sections above this one have?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -24,6 +24,16 @@ Please ensure your pull request (PR) adheres to the following guidelines:
...
+- If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
+  please document them in the `readme.html`.
+- Are you introducing any new cryptography modules? For what purpose?
+- If this is a new connector, or you are adding new actions
```
Comment (rgil-splunk) [inline .github/pull_request_template.md:31]:
```suggestion
- If this is a new connector or you are adding new actions
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -24,6 +24,16 @@ Please ensure your pull request (PR) adheres to the following guidelines:
 - [ ] Documentation
 - [ ] Other (please describe): 
 
+## Security Considerations (REQUIRED)
+- If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
+  please document them in the `readme.html`.
+- Are you introducing any new cryptography modules? For what purpose?
```
Comment (rgil-splunk) [inline .github/pull_request_template.md:30]:
```suggestion
- Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
```

**Review by rgil-splunk — APPROVED** [review]
unrelated to this PR but it seems we need to exclude `.github` in the lambda hooks. I swear I thought it was excluded but it looks like there are jobs running which make no sense for this repo 😅

**Comment by rgil-splunk** [comment]
also cc: @mattsayar-splunk for awareness since this is new user facing documentation

File: .github/pull_request_template.md (hunk 1 above)
Comment (pzhou-splunk) [reply .github/pull_request_template.md:27]: I think that would make sense, will add them

**Review by rgil-splunk — APPROVED** [review]

**Comment by rgil-splunk** [comment]
oh, I didn't realize this was the same PR until after I approved again because I clicked through the diff in the email link 😅

**Pull request merged**